Makefile

```diff
@@ -85,6 +85,11 @@ else
 	BUILDFLAGS := -buildmode=pie
 endif
 
+# whether stipping the binary
```

Comment:
Looking good. Can I ask we also set a default `no` at the top of the file, along with the other defaults, to make it easier to see what the set of build options are? Because, afaict, we don't have actual documentation on what the agent build options are ;-). So, add to this section:

```makefile
# Define if agent will be installed as init
INIT := no

# Set to "yes" if agent should support OpenTracing with http://jaegertracing.io.
TRACE := no
```

Reply:
Sure. will update it